zhaquirks/thirdreality/door_sensor_v2.py

```diff
@@ -0,0 +1,42 @@
+"""Third Reality door sensor devices."""
+
+from typing import Final
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.quirks.v2.homeassistant import UnitOfTime
+from zigpy.quirks.v2.homeassistant.number import NumberDeviceClass
+import zigpy.types as t
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class ThirdRealityDoorCluster(CustomCluster):
+    """Third Reality's door sensor private cluster."""
+
+    cluster_id = 0xFF01
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Define the attributes of a private cluster."""
+
+        open_delay_time: Final = ZCLAttributeDef(
+            id=0x0000,
+            type=t.uint16_t,
+            is_manufacturer_specific=True,
+        )
+
+
+(
+    QuirkBuilder("Third Reality, Inc", "3RDS17BZ")
+    .replaces(ThirdRealityDoorCluster)
+    .number(
+        attribute_name=ThirdRealityDoorCluster.AttributeDefs.open_delay_time.name,
+        min_value=0,
+        max_value=3600,
+        unit=UnitOfTime.SECONDS,
+        device_class=NumberDeviceClass.DURATION,
+        cluster_id=ThirdRealityDoorCluster.cluster_id,
```

Comment:
Just moving the `cluster_id` to another place here. Doesn't really matter, but I think we generally have attribute name, cluster id, and endpoint id as the first three options.
```suggestion
        attribute_name=ThirdRealityDoorCluster.AttributeDefs.open_delay_time.name,
        cluster_id=ThirdRealityDoorCluster.cluster_id,
        min_value=0,
        max_value=3600,
        unit=UnitOfTime.SECONDS,
        device_class=NumberDeviceClass.DURATION,
```
I'll apply this patch and merge the PR.